### PR TITLE
Update install script to install both flow and flow-c1

### LINKED ISSUE #1660: Master branch now uses `flow-c1` in install-scripts
The documentation for installing flow-cli refer to master branch install scripts, as do probably a lot of scripts and pipelines.

In a recent merge the renaming of `flow` to `flow-c1` was merged to master, breaking previous functionality and the desired interoperability

https://github.com/onflow/flow-cli/commit/b6c8838cfbbb9b59a504e2599cc9d4d182ff2096#diff-043df5bdbf6639d7a77e1d44c5226fd7371e5259a1e4df3a0dd5d64c30dca44fR130

### PR DESCRIPTION
**Pull request by chasefleming**
Closes #1660 

## Description

Install `flow` and `flow-c1` in install script.

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: install.ps1 [hunk 1]
```diff
@@ -38,6 +38,7 @@ $ErrorActionPreference = "Stop"
 $repo = "onflow/flow-cli"
 $versionURL = "https://api.github.com/repos/$repo/releases/latest"
 $assetsURL = "https://github.com/$repo/releases/download"
+$stableVersion = "v1.20.5" # The specific stable version to download
```
Comment (turbolent) [inline install.ps1:41]: Might be better to query the GitHub API and fetch the latest non pre-release (`prerelease == false`) using `Get-Version`

File: install.sh [hunk 2]
```diff
@@ -11,6 +11,8 @@ VERSION="$1"
 ARCH=""
 # The tag search term to use if no version is specified (first match is used)
 SEARCH_TERM="cadence-v1.0.0"
+# The specific version to download as "flow"
+FLOW_VERSION="v1.20.5"
```
Comment (turbolent) [inline install.sh:15]: Same as above for the PowerShell script above, maybe avoid hard-coding a specific release

**Review by turbolent — COMMENTED** [review]
Good idea to just install both versions!

File: install.ps1 (hunk 1 above)
Comment (chasefleming) [reply install.ps1:41]: I tried making the change, but unfortunately, grep prerelease false doesn't return the correct entry. Also, we are trying to code freeze this last version now, so I don't think hardcoding is too bad. I agree the other way is more ideal, but to be honest, I'm a little out of my depth here with getting this and the corresponding PowerShell script to work. However, the current hardcoded version scripts seem to work. Feel free to take a shot if you think you can get it to work without hardcoding.

Comment (turbolent) [reply install.ps1]: Improved the PowerShell script in [bdf33f5](https://github.com/onflow/flow-cli/pull/1663/commits/bdf33f5bb6148e24b995901203c8aeb256a996bc).

Will improve the Shell script now

File: install.sh (hunk 2 above)
Comment (turbolent) [reply install.sh]: Improved this in [0fa235d](https://github.com/onflow/flow-cli/pull/1663/commits/0fa235da235ee62241ce58955bfe74130c33f72e)

File: install.ps1 (hunk 1 above)
Comment (turbolent) [reply install.ps1]: Updated the shell script in [0fa235d](https://github.com/onflow/flow-cli/pull/1663/commits/0fa235da235ee62241ce58955bfe74130c33f72e)

**Review by turbolent — APPROVED** [review]

**Review by janezpodhostnik — APPROVED** [review]

**Pull request merged**